src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/integration/PrisonersIntegrationTest.kt

```diff
@@ -17,6 +17,17 @@ class PrisonersIntegrationTest : IntegrationTestBase() {
       .expectStatus().isOk
       .expectHeader().contentType("application/json")
       .expectBody().json(expectedOutput, true)
+
+    // Reset mocks to ensure it uses the cache
+    prisonerSearchApiMockServer.resetAll()
```

Comment:
I prefer the style of doing `verify(prisonerSearchApiMockServer, times(1))..stubGetPrisonersList(prisonId, "", 500, 0, 200))` instead of removing the stub, just because the test failure would be more clear. 
Not a big deal though, this is good enough. I just wanted to share the idea

Reply:
Agree this is better but I'll add it to a new ticket to enhance later.